code/game/gamemodes/personal_objectives/abyssor/splash_water.dm

```diff
@@ -0,0 +1,38 @@
+/datum/objective/abyssor_splash
+	name = "Splash Water"
+	triumph_count = 0
```

Comment:
Why isn't this value used?

Reply:
Ah, thats a leftover when I first started doing objectives independent of the antag datum. I made all personal objectives use this var for awarding triumphs now.